packages/ui/src/signals/labelClip.ts

```diff
@@ -0,0 +1,11 @@
+import {signal} from '@preact/signals';
+
+interface LabelClipDrag {
+  left: number;
+  dragging: boolean;
+}
+
+export const labelClipDragging = signal<LabelClipDrag>({
```

Comment:
The name of the signal should be suffixed with `Signal` to match other signals we use.